MarsSpecFlowProject/MarsSpecFlowProject/Page/ProfilePage.cs

```diff
@@ -0,0 +1,201 @@
+﻿
+ using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TechTalk.SpecFlow;
+using MarsSpecFlowProject.Utils;
+using NUnit.Framework;
+using SeleniumExtras.PageObjects;
+
+namespace MarsSpecFlowProject.Page
+{
+    class ProfilePage:BasePage
+    {
```

Comment:
hey Reshma you still have not implemented the POM pattern, please implement this here is an example 
https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html